python/src/aiconfig/AIConfigSettings.py

```diff
@@ -218,6 +218,29 @@ def get_model_name(self, prompt: Union[str, Prompt]):
         else:
             # Expect a ModelMetadata object
             return prompt.metadata.model.name
+    
+    def set_default_model(self, model_name: Union[str, None]):
+        """
+        Sets the model to use for all prompts by default in the AIConfig. Set to None to delete the default model.
+
+        Args:
+            model_name (str): The name of the default model.
+        """
+        self.metadata.default_model = model_name
+    
+    def set_model_parser(self, model_name: str, model_parser_id: Union[str, None]):
+        """
```

Comment:
Should this be synced to the ModelParserRegistry? Just thought of this while implementing the python side.

ex:
config.set_model_parser("gpt-3.5-turbo", "HuggingFaceTextParser") would update the the config but user would then have to manually run
ModelParserRegistry.register_model_parser

Could also make sense to make ModelParserRegistry a non static class and have it be an attribute of AIConfigRuntime

Reply:
Hmm, good catch. Maybe yes -- add to this and to TS at the top of the stack?

Reply:
Will update on top of stack